### PR TITLE
Add Content usable and WCAG 2.2 SCs

### PR DESCRIPTION
**Pull request by SteveALee**
Should be done ASAP as we pointed people to this page in AccessU

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -77,6 +77,20 @@ W3C is actively working to provide additional guidance on cognitive accessibilit
...
+
+- people with learning and cognitive disabilities,
+- aims and objectives for usable content,
+- design patterns (ways) to make content usable,
+- including users in design and testing activities, and
+- personas (examples) and user needs.
+ 
+ ### Potential Additional Requirements in Standards
+ 
+WCAG 2.2 is currently being develeped and includes several potential “success criteria” with an emphasis on cognitive accessibility. These may become part of the final publication, depending on the outcome of the public Wide Review process that W3C standards undergo.
```
Comment (SteveALee) [inline index.md:92]: This content is time sensitive. I'm not sure how that is handled or if it is even OK?

**Comment by SteveALee** [comment]
Eek - the preview builds is failing due to a failure to find a Jekyll component. How is this maintained now Yatil is not longer managing?

**Comment by slhenry** [comment]
Hi Steve. First question is do we really want to make such changes to this page?

Two options for approaches:

1. Add info about drafts-in-progress to this page.
Pro: Users who land on this page and want to see the drafts, can get to that info directly.
Con: Another place for us to update info (which we tend to forget with publications).
Con: Adds more content to this page.

2. Point elsewhere for drafts-in-progress.
Pro: Less content on this page, and much less need to update it.
Con: Users interested in drafts have to follow another link to get to them.

So far we've gone with approach 2. I'm a bit hesitant to move to 1.

I did just now make a few tweaks to help this stand out:
**Resources in development are listed in the [COGA wiki main page](@@).**
**Resources in development are introduced in the [Personalization Overview](@@).**

If you think we should change to Approach 1, or have other ideas for making the drafts easier to know about about and get to for Approach 2, let's talk briefly to get in synch on what info we want to include.

**Comment by SteveALee** [comment]
Good point and I'm certainly OK with that for wcag. Hence my comment.

But content usable is a note and so not subject to the full w3c process. As it's not even the first working draft its not clear it matters where it is in its own standard cycle. Thus I'm personally happy for something specifically mentioning it.

As long as there's a clear link to follow from the page to the cibtent usable im happy.

Normally I'd not want to point to a wiki as they are unstable but tgmhis is a closed wide so that's not a problem.

Thus I think your change is good for now. I assume you pushed? I was alarmed to see the preview build broke. Normally the main deploy is pretty much identical..

Thanks for responding and I don't wish to take more of you time before Tuesday.

**Comment by nitedog** [comment]
@slhenry this seems inconsistent with the Personalization and Pronounciation pages where we list and describe on-going work. I kind of prefer option 1 to make our work more visible to the public.

**Comment by SteveALee** [comment]
>  I kind of prefer option 1 to make our work more visible to the public.

I was going to say I will take this to the coga TF but I think it needs wider discussion as there could be a principle here.

**Comment by SteveALee** [comment]
I'm happy with the [changes commited directly  by @slhenry](8951f98efb586dc38782611b40cb6e8efd9ee541) (thanks)

Have requested TF review the page.

**Pull request closed without merging**